### PR TITLE
SAIC-156: Fix DHCP issue for all tenants

### PR DESCRIPTION
**Pull request by miarmak**
Since the creation order of DHCP port and instance port can be random,
sometimes, the DHCP port can take the IP already used by instance on the
source could. To avoid it, this fix will disable DHCP for all networks
in all tenants, participated im migration process, and free it's IP
before instance port creation, then re-enable the DHCP.

### CODE REVIEW

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.

File: cloudferrylib/os/actions/prepare_networks.py [hunk 1]
```diff
@@ -31,12 +31,17 @@ def run(self, info=None, **kwargs):
         keep_ip = self.cfg.migrate.keep_ip
 
         instances = info_compute[utl.INSTANCES_TYPE]
+
+        # Get all tenants, participated in migration process
+        tenants = set([])
```
Comment (oleksii-shyman) [inline cloudferrylib/os/actions/prepare_networks.py]: why do we need that brackets?
set() works

**Comment by oleksii-shyman** [comment]
looks ok + 1

File: cloudferrylib/os/actions/prepare_networks.py (hunk 1 above)
Comment (miarmak) [inline cloudferrylib/os/actions/prepare_networks.py]: Just a habit of declaring the empty set) Will fix it.

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.

**Comment by roman-verchikov** [comment]
+1

**Pull request merged**